taxonomies/labels.txt

```diff
@@ -11,22 +11,26 @@ stopwords:sv:certifierad, ingrediens, godkänd, råvara, enlighet, med, kriterie
 synonyms:en:coloring, colouring, color, colour
 synonyms:ru:окраска, цвет
 synonyms:fi:väri, värjäys, väriaine
+synonyms:fr:colorant, couleur
```

Comment:
We should add synonyms here only if we find labels that have those synonyms. e.g. in French we can see "sans colorants", but it's never translated to "sans couleurs", so we don't need this line.